### PR TITLE
@alloy => Icon fullscreen, minor QA

### PR DESCRIPTION
**Pull request by eessex**
- Adds an Icon for fullscreen image UI
- Removes header class selectors & replaces with styled component references
- Sets lead_paragraph width to 100%

### CODE REVIEW

File: src/components/publishing/header/feature_header.tsx [hunk 1]
```diff
@@ -215,15 +215,15 @@ const FeatureHeaderContainer = Div.extend`
   height: 100vh;
   &[data-type='text'] {
     height: auto;
-    .feature__title {
+    ${Title} {
```
Comment (alloy) [inline src/components/publishing/header/feature_header.tsx]: Were these referring to the same thing but previously in a roundabout way?

**Comment by alloy** [comment]
LGTM 👍

File: src/components/publishing/header/feature_header.tsx (hunk 1 above)
Comment (eessex) [reply src/components/publishing/header/feature_header.tsx]: Yeah, they are the same div

Comment (alloy) [reply src/components/publishing/header/feature_header.tsx]: Ah ok, so where was that classname coming from? Is that what styled-components generated?

Comment (eessex) [reply src/components/publishing/header/feature_header.tsx]: Originally they came from the [header component](https://github.com/artsy/reaction/blob/master/src/components/publishing/header/header.tsx), the title class has already been removed.  The deck will probably be removed too in a future PR.

Comment (alloy) [reply src/components/publishing/header/feature_header.tsx]: Gotcha, thanks 👍

**Pull request merged**